dashboard/src/actions/comparisonActions.js

```diff
@@ -111,11 +158,9 @@ export const parseChartData = () => (dispatch, getState) => {
     }, Object.create(null));
 
     for (const [key, value] of Object.entries(result)) {
-      console.log(key);
-
       const map = {};
       for (const element of value) {
-        map[element.name] = element.time_taken.trim();
+        map[element[chartObj.keyToParse]] = element[chartObj.valueToGet].trim();
       }
```

Comment:
Alternately,
```javascript
      const map = value.reduce((a, c) => {
        a[c[chartObj.keyToParse]] = c[chartObj.valueToGet].trim();
        return a;
        }, {});
```
Hmmm, that's actually not quite wonderful...how about
```javascript
      const map = {};
      value.forEach((e) => {
        map[e[chartObj.keyToParse]] = e[chartObj.valueToGet].trim();
        });
```
Bummer...that's better, but not a huge improvement, either...nevermind.  😉